homeassistant/components/shelly/climate.py

```diff
@@ -526,3 +532,106 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         await self.call_rpc(
             "Thermostat.SetConfig", {"config": {"id": self._id, "enable": mode}}
         )
+
+
+class RpcBluTrvClimate(ShellyRpcEntity, ClimateEntity):
+    """Entity that controls a thermostat on RPC based Shelly devices."""
+
+    _attr_max_temp = RPC_THERMOSTAT_SETTINGS["max"]
+    _attr_min_temp = RPC_THERMOSTAT_SETTINGS["min"]
+    _attr_supported_features = (
+        ClimateEntityFeature.TARGET_TEMPERATURE
+        | ClimateEntityFeature.TURN_OFF
+        | ClimateEntityFeature.TURN_ON
+    )
+    _attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
+    _attr_target_temperature_step = RPC_THERMOSTAT_SETTINGS["step"]
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+
+    def __init__(self, coordinator: ShellyRpcCoordinator, id_: int) -> None:
+        """Initialize."""
+
+        super().__init__(coordinator, f"{BLU_TRV_IDENTIFIER}:{id_}")
+        self._id = id_
+        self._config = coordinator.device.config[f"{BLU_TRV_IDENTIFIER}:{id_}"]
+        ble_addr: str = self._config["addr"]
+        self._attr_unique_id = f"{ble_addr}-{self.key}"
+        name = self._config["name"] or f"shellyblutrv-{ble_addr.replace(":", "")}"
+        self._attr_name = name
```

Comment:
According to https://developers.home-assistant.io/docs/core/entity/#entity-naming we should set this to `None`
```suggestion
        self._attr_name = None
```
Note that we need to do this constructor otherwise it will be replaced by our base class naming logic.